obal/data/modules/srpm.py

```diff
@@ -0,0 +1,122 @@
+"""
+Build SRPM
+"""
+
+import shutil
+import os
+import subprocess
+from tempfile import mkdtemp
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.obal import chdir # pylint:disable=import-error,no-name-in-module
+
+def run_command(command):
+    """
+    Run a system command
+    """
+    return subprocess.check_output(
+        ' '.join(command),
+        universal_newlines=True,
+        shell=True,
+        stderr=subprocess.STDOUT
+    )
+
+
+def copy_sources(spec_file, package_dir, sources_dir):
+    """
+    Copy RPM sources to rpmbuild environment
+    """
+    command = ["spectool", "--list-files", spec_file]
+    sources = run_command(command)
+
+    for source in sources.split("\n"):
+        if not source:
+            continue
+
+        source_link = source.split(' ')[1]
+
+        if not source_link.startswith('http'):
+            shutil.copy(os.path.join(package_dir, source_link), sources_dir)
+
+    with chdir(package_dir):
+        run_command(["git-annex", "lock"])
+        annexed_files = run_command(["git-annex", "find", "--include='*'"])
```

Comment:
```suggestion
        annexed_files = run_command(["git-annex", "find", "--include", "*"])
```

that plus the suggestions from @ekohl make it work with no `join` and no `shell` for me

Reply:
FWIW, this is the patch I have applied locally:

```patch
% git diff
diff --git obal/data/modules/srpm.py obal/data/modules/srpm.py
index 6255fda..e7d8fbf 100644
--- obal/data/modules/srpm.py
+++ obal/data/modules/srpm.py
@@ -15,9 +15,8 @@ def run_command(command):
     Run a system command
     """
     return subprocess.check_output(
-        ' '.join(command),
+        command,
         universal_newlines=True,
-        shell=True,
         stderr=subprocess.STDOUT
     )
 
@@ -40,7 +39,7 @@ def copy_sources(spec_file, package_dir, sources_dir):
 
     with chdir(package_dir):
         run_command(["git-annex", "lock"])
-        annexed_files = run_command(["git-annex", "find", "--include='*'"])
+        annexed_files = run_command(["git-annex", "find", "--include", "*"])
         annexed_files = annexed_files.splitlines()
 
         if not annexed_files:
@@ -87,15 +86,15 @@ def main():
         shutil.copy(spec_file, base_dir)
 
         command = ['rpmbuild', '-bs']
-        command += ['--define', '"_topdir %s"' % base_dir]
-        command += ['--define', '"_sourcedir %s"' % sources_dir]
-        command += ['--define', '"_builddir %s"' % build_dir]
-        command += ['--define', '"_srcrpmdir %s"' % base_dir]
-        command += ['--define', '"_rpmdir %s"' % base_dir]
+        command += ['--define', '_topdir %s' % base_dir]
+        command += ['--define', '_sourcedir %s' % sources_dir]
+        command += ['--define', '_builddir %s' % build_dir]
+        command += ['--define', '_srcrpmdir %s' % base_dir]
+        command += ['--define', '_rpmdir %s' % base_dir]
         command += ['--undefine', 'dist']
 
         if scl:
-            command += ['--define', '"scl %s"' %  scl]
+            command += ['--define', 'scl %s' %  scl]
 
         command += [os.path.join(base_dir, '%s.spec' % os.path.basename(package))]
 
```